commit 63126f874bfcae3cafc11051fd33b6a5d938718f (refs/changes/43/543/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-28T17:43:10-07:00 (3 years ago)
    
    DOCKER-938 Update sdcnode version for OpenSSL 1.0.2j/1.0.1u vulnerability fixes
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/Makefile b/Makefile
index 0a24425..6a41a5f 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ JSSTYLE_FLAGS	 = -f tools/jsstyle.conf
 SMF_MANIFESTS_IN = smf/manifests/docker.xml.in
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v0.10.46
+NODE_PREBUILT_VERSION=v0.10.47
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zoneecdh
 	NODE_PREBUILT_IMAGE=de411e86-548d-11e4-a4b7-3bb60478632a
